### ISSUE TITLE
Split files

### ISSUE THREAD

**Issue by Tom-Johnston** [jstockwin/Film-Night#23]
- Move the JS into separate files that are either linked or included.
- Split the styles.

**Comment by jstockwin** [comment]
Has this been done in e.g. 5a0a377636731d1fdc6ab75a506ca861b4d97d77 so that this issue can be closed?

**Comment by Tom-Johnston** [comment]
No. 5a0a377 was organising files although I am not sure it is complete; I only did some to fill the 15 minutes before hall.
